src/main/java/Domain/ExternalServices/SupplyService/AdapterSupply.java

```diff
@@ -1,5 +1,17 @@
 package Domain.ExternalServices.SupplyService;
 
-public class AdapterSupply {
+import Domain.ExternalServices.ExternalService;
+
+public class AdapterSupply implements ExternalService{
+
+    public AdapterSupply() {
+    
+    }
+
+    @Override
+    public boolean ConnectToService() {
+        // Connect to the supply service
```

Comment:
the addapter wants to call the implement and try to connect - tal is doing that?
if so- leave him //TODO pls